src/components/plots/helpers/calculateVolcanoDataPoints.js

```diff
@@ -33,18 +33,15 @@ const calculateVolcanoDataPoints = (config, data) => {
       const { logFC } = datum;
       const p_val_adj = parseFloat(datum.p_val_adj);
 
-      const pvalueThreshold = (
-        10
-        ** (-1 * config.negLogpValueThreshold)
-      ).toExponential(3);
+      const { adjPvalueThreshold } = config;
```

Comment:
Should this be `config.adjPvalueThreshold`

Reply:
`const { adjPvalueThreshold } = config;` is a [destructing assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) so it is just syntactic butter for `const adjPvalueThreshold = config.adjPvalueThreshold`

Reply:
Nice! Thanks.